docs/design-proposals/cloud-edge-communication/cloud-edge-application-communication-framework.md

```diff
@@ -0,0 +1,17 @@
+# Arktos Cloud-edge Application Communication Framework Overview
```

Comment:
This is the initial idea for the feature proposal.  Per the discussion in community meeting, I've started a google doc with this initial content. Let's do collaboratively refine the document and add detailed use cases and requirements.

https://docs.google.com/document/d/19Dt14EfpB7aO4eJwd8DxBWv87pcJ92x4XKxz_jRdkG4/edit?usp=sharing

Reply:
@XiaoningDing, provided feedback in the GoogleDoc.

Reply:
@XiaoningDing, also channel level encryption based (VPN) are not the right solution. Application level mutual TLS based authentication to go along with OPA like access control is the way to go. I did some work with India team some time ago on solving this problem for edge cloud using SPIFFEE/SPIRE. That is the right way of addressing this edge cloud issue. We ended up working with the SPIFFEE/SPIRE community to provide support for multiple SPIRE servers in a nested hierarchy.

IMO, this is the right way to do it at the application level versus using the current networking channel based approach. We did a very detail design for all this. I need to dig that out.